themes/Makefile

```diff
@@ -0,0 +1,16 @@
+BASE_THEME	= yeti
+THEMES_DIR	= themes
+OUTPUT_DIR	= ../inginious/frontend/static/css/themes
+
+compile:
+	if [ ! -d "node_modules" ]; then make install; fi
```

Comment:
I actually don't ask this to be modified, but `make` supports dependencies handling. That's one of his core features.